pkg/controllers/hostedcluster/hostedcluster.go

```diff
@@ -73,6 +76,17 @@ func InstallControllers(stopCh <-chan struct{}, aggregatorConfig *aggregatorapis
 		mgr, err := ctrl.NewManager(restConfig, ctrl.Options{
```

Comment:
I am OK for this PR. I will merge it soon.

Can we reuse the control-manager in controller.go instead of creating a new one?

Reply:
the manager for hosted cluster is using management cluster kubeconfig, while the manager other controllers in controlplane is using contrplane's kubeconfig.